apps/communications/dialer/js/mmi.js

```diff
@@ -356,19 +356,20 @@ var MmiManager = {
             };
             var conn = navigator.mozMobileConnections[cardIndex || 0];
             var operator = MobileOperator.userFacingInfo(conn).operator;
-            var title = self.prependSimNumber(operator ? operator : '',
+            var titleL10n = self.prependSimNumber(operator ? operator : '',
```

Comment:
nit: Please move the arguments to the line below, that'll make the formatting nicer, e.g.

```javascript
var titleL10n = self.prependSimNumber(
  operator ? operator : '', cardIndex
);
```